### PR TITLE
Add llite statistics

### PR DESCRIPTION
**Pull request by RDruon**
Add llite stats

### CODE REVIEW

**Comment by spoutn1k** [comment]
This is super useful when runnning rust scripts on a client, what is preventing the merge ?

File: src/llite/mod.rs [hunk 1]
```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2023 DDN. All rights reserved.
```
Comment (jgrund) [inline src/llite/mod.rs:1]:
```suggestion
// Copyright (c) 2024 DDN. All rights reserved.
```

File: src/ldlm/mod.rs [hunk 2]
```diff
@@ -22,5 +22,7 @@ where
     I: Stream<Token = char>,
     I::Error: ParseError<I::Token, I::Range, I::Position>,
 {
-    (string(LDLM), period()).with(ldlm_namespace_parser::parse().or(ldlm_service_parser::parse()))
+    (attempt(string(LDLM)), period())
```
Comment (jgrund) [inline src/ldlm/mod.rs:25]: Why is this wrapped in `attempt` ?

File: src/llite/mod.rs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+enum LliteStat {
+    Stats(Vec<Stat>),
+}
+
+fn llite_stat<I>() -> impl Parser<I, Output = (Param, LliteStat)>
+where
+    I: Stream<Token = char>,
+    I::Error: ParseError<I::Token, I::Range, I::Position>,
+{
+    choice(((param(STATS), stats().map(LliteStat::Stats)).message("while parsing llite_stat"),))
```
Comment (jgrund) [inline src/llite/mod.rs:44]:
```suggestion
    choice(((param(STATS), stats().map(LliteStat::Stats)).message("while parsing llite_stat")))
```

File: src/fixtures/valid/llite_client.txt
Comment (jgrund) [inline src/fixtures/valid/llite_client.txt:1]: This needs to be inside a few new dirs:

`fixtures/lustre-X-Y-Z/client/

File: src/ldlm/mod.rs (hunk 2 above)
Comment (RDruon) [reply src/ldlm/mod.rs:25]: No need to, I removed it

File: src/llite/mod.rs (hunk 3 above)
Comment (RDruon) [reply src/llite/mod.rs]: No need for `choice`, I removed it

**Review by utopiabound — APPROVED** [review]

**Pull request merged**